mobile/src/main/java/org/hawkular/client/android/fragment/TriggerDetailFragment.java

```diff
@@ -16,33 +16,90 @@
  */
 package org.hawkular.client.android.fragment;
 
+import org.hawkular.client.android.Animae.MyBounceInterpolator;
 import org.hawkular.client.android.R;
 import org.hawkular.client.android.backend.model.Trigger;
 import org.hawkular.client.android.util.Fragments;
+import org.jboss.aerogear.android.store.DataManager;
+import org.jboss.aerogear.android.store.generator.IdGenerator;
+import org.jboss.aerogear.android.store.sql.SQLStore;
+import org.jboss.aerogear.android.store.sql.SQLStoreConfiguration;
 
 import android.app.Fragment;
+import android.content.Context;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+import android.widget.ImageButton;
+import android.widget.TextView;
 import android.widget.Toast;
+
+import java.util.UUID;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
 import icepick.State;
 
 
 public class TriggerDetailFragment extends android.support.v4.app.Fragment {
             @State
             @Nullable
             Trigger trigger;
+    @BindView(R.id.add_button)
+    ImageButton button;
```

Comment:
Let's avoid to use generic names like _button_. Let's use something that really represent button action like _addButton_